gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
```

Comment:
Make sure this file is NOT tracked by .git by including it in your .gitignore as per the instructions above.